api/swagger.yaml

```diff
@@ -9316,6 +9337,11 @@ paths:
             - `label=<service label>`
             - `mode=["replicated"|"global"]`
             - `name=<service name>`
+        - name: "status"
+          in: "query"
+          type: "boolean"
+          description: "Include service status, with count of running and desired tasks"
+          default: false
```

Comment:
Wonder if this is what's causing the panic. I guess a query-string may not support a "default" value? https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-6.2

> This keyword can be used to supply a default JSON value associated with a particular schema.  It is RECOMMENDED that a default value be  valid against the associated schema.

Reply:
Perhaps `"required": false` would achieve the same

Reply:
Thanks for looking into this @dperny